packages/core/package.json

```diff
@@ -58,6 +58,7 @@
     "test:e2e": "E2E=true jest --runInBand --detectOpenHandles --passWithNoTests"
   },
   "dependencies": {
+    "@aws-lambda-powertools/parameters": "^2.1.0",
```

Comment:
Since we don't use it directly on the code from Core, only on the API, do we really need to add it here?